app/main/api/handlers/newMnemonics.ts

```diff
@@ -81,7 +81,7 @@ function buildSuccess(mnemonics: Mnemonics): NewMnemonicsSuccess {
  * @param {t.LiteralType<NewMnemonicsErrors>} error
  * @returns {NewMnemonicsError}
  */
-function buildError(error: t.LiteralType<NewMnemonicsErrors>): NewMnemonicsError {
+export function buildError(error: t.LiteralType<NewMnemonicsErrors>): NewMnemonicsError {
```

Comment:
This function (and the equivalent one in app/main/api/handlers/importSeed) is being exported from main and imported in renderer. Perhaps, their place should be the common folder. In app/common/runtimeTypes/ipc/wallets.ts (buildGetWalletError), there is an example of such a factory function (though perhaps it would be great to generalize it to build any type of Error).

Reply:
I think that this should be done in another PR solving their own issue

Reply:
That's issue #327 now :wink: